lib/std/os/linux/io_uring.zig

```diff
@@ -3491,3 +3788,340 @@ test "accept multishot" {
         os.closeSocket(client);
     }
 }
+
+test "accept/connect/send_zc/recv" {
+    try skipKernelLessThan(.{ .major = 6, .minor = 0, .patch = 0 });
+
+    var ring = try IO_Uring.init(16, 0);
+    defer ring.deinit();
+
+    const socket_test_harness = try createSocketTestHarness(&ring);
+    defer socket_test_harness.close();
+
+    const buffer_send = [_]u8{ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 0xa, 0xb, 0xc, 0xd, 0xe };
+    var buffer_recv = [_]u8{0} ** 10;
+
+    // zero-copy send
+    const send = try ring.send_zc(0xeeeeeeee, socket_test_harness.client, buffer_send[0..], 0, 0);
+    send.flags |= linux.IOSQE_IO_LINK;
+    _ = try ring.recv(0xffffffff, socket_test_harness.server, .{ .buffer = buffer_recv[0..] }, 0);
+    const submitted = try ring.submit();
+    if (submitted != 2) return error.SkipZigTest; // on kernel 5.8 (without zc support)
```

Comment:
Should this case invoke skipKernelLessThan() so the test doesn't hide a failure if something goes wrong?  (E.g., if if `ring.submit` is broken, you want to fail the test?)

Reply:
Uf, great point. 
That is leftover while I was fighting with different kernel behaviors.
Will remove that, it is covered with skipKernelLessThan now.